bin/citrea/tests/evm/mod.rs

```diff
@@ -31,23 +31,22 @@ async fn web3_rpc_tests() -> Result<(), anyhow::Error> {
     let storage_dir = tempdir_with_children(&["DA", "sequencer", "full-node"]);
     let da_db_dir = storage_dir.path().join("DA").to_path_buf();
     let sequencer_db_dir = storage_dir.path().join("sequencer").to_path_buf();
-    let da_db_dir_cloned = da_db_dir.clone();
 
     let (port_tx, port_rx) = tokio::sync::oneshot::channel();
+
+    let rollup_config =
+        create_default_rollup_config(true, &sequencer_db_dir, &da_db_dir, NodeMode::SequencerNode);
+    let sequener_config = SequencerConfig {
+        min_soft_confirmations_per_commitment: DEFAULT_MIN_SOFT_CONFIRMATIONS_PER_COMMITMENT,
```

Comment:
ok now I see this is to set to 1000.

for most tests it doesn't matter what this value is set to.

First try the tests with just SequencerConfig::deafult()

if any fails, rename this constant to `TEST_SEND_NO_COMMITMENT_MIN_SOFT_CONFIRMATIONS_PER_COMMITMENT`

haha

Reply:
hahah got it